src/cli/options/options.go

```diff
@@ -106,6 +111,18 @@ func AddDisableIncrementalsFlag(cmd *cobra.Command) {
 	cobra.CheckErr(fs.MarkHidden(DisableIncrementalsFN))
 }
 
+// Adds the hidden '--disable-delta' cli flag which, when set, disables
+// delta based backups.
+func AddDisableDeltaFlag(cmd *cobra.Command) {
```

Comment:
pitch, instead of adding this (and in the future, more --disable-foo) flags, how about a single `--disable` flag with a string slice arg?  Ex:
```go
	fs.BoolVar(
		&disableFV,
		DisableFN,
		false,
		"Disable specific functionality; accepts any of [incrementals, delta]")
```

This way we have one easily extensible flag instead of a handful of bools.

Reply:
Hmm, interesting idea. I am wondering if we should be reserving `--disable` and `--enable` for non internal flags though.